src/main/locales/cy/translation.json

```diff
@@ -1,5 +1,84 @@
 {
+  "template": {
+    "serviceName": "Gwnewch gais i fabwysiadu plentyn",
+    "phaseBanner": {
+      "phase": "Alffa",
+      "content": "Mae hwn yn wasanaeth newydd - eich <a href='#'> adborth </a> yn ein helpu i'w wella."
+    },
+    "links": {
+      "profile": "Proffil",
+      "signOut": "arwyddo Allan"
+    }
+  },
   "home": {
     "title": "Helo Byd!"
+  },
+  "error": {
+    "heading": "Aeth rhywbeth o'i le"
+  },
+  "pageNotFound": {
+    "heading": "Tudalen heb ei darganfod"
+  },
+  "taskList": {
+    "title": "Gwnewch gais i fabwysiadu plentyn sydd wedi'i roi yn eich gofal",
+    "taskItems": {
+      "applicationDetails": {
+        "heading": "Ychwanegwch fanylion y cais",
+        "subTasks": {
+          "numOfChildren": "Nifer y plant a'r ymgeiswyr",
+          "chooseFamilyCourt": "Dewiswch eich llys teulu"
+        }
+      },
+      "yourDetails": {
+        "heading": "Ychwanegwch eich manylion",
+        "subTasks": {
+          "personalDetails": "Eich manylion personol",
+          "contactDetails": "Eich manylion cyswllt",
+          "relationshipDetails": "Manylion eich perthynas",
+          "relationshipDetailsOnlyApplicant": "Eich perthynas (pe bai'r defnyddiwr yn dweud mai nhw oedd yr unig ymgeisydd)",
+          "uploadDocuments": "Llwythwch i fyny eich dogfennau adnabod"
+        }
+      },
+      "childDetails": {
+        "heading": "Ychwanegwch fanylion y plentyn",
+        "subTasks": {
+          "birthCertficate": "Manylion eu tystysgrif geni",
+          "adoptionCertificate": "Manylion y dystysgrif fabwysiadu",
+          "history": "Eu hanes gyda chi",
+          "placementOrderDetails": "Manylion eu gorchymyn lleoliad",
+          "birthMotherDetails": "Manylion eu mam enedigol",
+          "birthFatherDetails": "Manylion eu tad biolegol neu riant arall",
+          "previousCourtOrder": "Gorchymyn llys blaenorol ar gyfer y plentyn",
+          "courtOrderDetails": "Manylion gorchymyn llys ar gyfer unrhyw frodyr a chwiorydd neu hanner brodyr a chwiorydd"
+        }
+      },
+      "adoptionContacts": {
+        "heading": "Ychwanegwch y cysylltiadau mabwysiadu",
+        "subTasks": {
+          "socialWorker": "Manylion eu tystysgrif geni",
+          "childsSocialWorker": "Manylion y dystysgrif fabwysiadu",
+          "solictor": "Eu hanes gyda chi"
+        }
+      },
+      "declarePayments": {
+        "heading": "Datgan taliadau",
+        "subTasks": {
+          "declarePayments": "Datgan unrhyw daliadau a wnaed neu a dderbyniwyd"
+        }
+      },
+      "reviewApplication": {
+        "heading": "Adolygu cais, talu ac anfon",
+        "subTasks": {
+          "reviewApplication": "Adolygu cais, talu ac anfon"
+        },
+        "alert": {
+          "content": "Bydd angen eich cerdyn credyd neu ddebyd arnoch chi"
+        }
+      }
+    },
+    "taskStatus": {
+      "complete": "Wedi'i gwblhau",
+      "notStarted": "Heb ddechrau"
+    }
```

Comment:
I didn't know you know welsh :) but i don't think we should do this.. There is welsh language translate team who will do this for us

Reply:
Byddaf yn dadwneud ;)